packages/replace/README.md

```diff
@@ -52,6 +52,25 @@ The configuration above will replace every instance of `__buildEnv__` with `'pro
 
 Typically, `@rollup/plugin-replace` should be placed in `plugins` _before_ other plugins so that they may apply optimizations, such as dead code removal.
 
+The most popular case is replacing process.env.NODE_ENV with development or production environment.
+
+```js
+import replace from '@rollup/plugin-replace';
+
+export default {
+  input: 'src/index.js',
+  output: {
+    dir: 'output',
+    format: 'cjs'
+  },
+  plugins: [
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('production') // JSON.stringify('development')
```

Comment:
```suggestion
      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
```

Will it be more universal way?

Reply:
I usually use config array and specify all values explicitly. NODE_ENV may raise additional question how to pass it to rollup.

Reply:
I took the best of both your comments to add a comment to the code explaining alternative values. hopefully that will be clear for people who take both approaches.

Reply:
Good